### PR TITLE
Minor tweaks to SynDrones

### PR DESCRIPTION
**Pull request by Ziiro**
:cl:
tweak: SynDrones now spawn with a syndie duffel bag. Inside of it is a Stretchkin pistol, in addition to their uplink.
/:cl:

### CODE REVIEW

**Comment by Ziiro** [comment]
Balance Concerns:

I will likely have to adjust down the amount of TC given to the drones, slightly. _Maybe_ remove the knife and stretchkin, considering how easily they can source a *real* weapon. A silenced stretchkin doesn't really break the bank on TC, though. Throwing in the encryption key seems mostly necessary - you're trading all of your TC for a fragile partner to help you. The least you can get out of it is a way to consistently communicate with them.

**Comment by SamHPurp** [comment]
I strongly dislike anything to do with drones being antaggy again.

Really unkeen.

**Comment by Ziiro** [comment]
This is more like a holopara.

Plus, they're pretty visually distinct:

dreamseeker_2018-01-13_16-57-12

**Comment by Ziiro** [comment]
Removed them from uplinks

Just left the tweaks to their starting inventory in.

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -367,17 +367,17 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
...
- 	player_minimum = 25
- 
+	name = "Holoparasites"
+	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
+			organic host as a home base and source of fuel."
+	item = /obj/item/storage/box/syndie_kit/guardian
+	cost = 14
+	exclude_modes = list(/datum/game_mode/nuclear)
+	player_minimum = 25
+
```
Comment (Ziiro) [inline code/modules/uplink/uplink_item.dm]: There were spaces here. I removed them.

**Pull request closed without merging**